src/schema/datatypes/anat.yml

```diff
@@ -0,0 +1,44 @@
+group1:
```

Comment:
it feels we would just want a list here instead of coming up with some `groupX` key.

Reply:
Continued in new comment thread tied to `anat.yaml` at https://github.com/bids-standard/bids-specification/pull/475#discussion_r428977646.